packages/applications-service-api/__tests__/unit/repositories/interestedParty.repository.test.js

```diff
@@ -0,0 +1,72 @@
+const {
+	createInterestedParty
+} = require('../../../src/repositories/interestedParty.ni.repository');
+
+const mockInterestedPartyCreate = jest.fn();
+jest.mock('../../../src/models', () => {
+	return {
+		InterestedParty: {
+			create: (data) => mockInterestedPartyCreate(data)
+		}
+	};
+});
+
+describe('interestedParty NI repository', () => {
+	describe('createInterestedParty', () => {
+		it('calls create and returns data', async () => {
+			const input = { some: 'data' };
+			mockInterestedPartyCreate.mockResolvedValueOnce({
+				dataValues: {
+					id: 1234,
+					...input
+				}
+			});
+
+			const result = await createInterestedParty(input);
+
+			expect(mockInterestedPartyCreate).toHaveBeenCalledWith(input);
+			expect(result).toEqual({
+				id: 1234,
+				...input
+			});
+		});
+
+		describe('sequelize throws error', () => {
+			it("throws apierror if sequelize 'wrong value' error is thrown", async () => {
```

Comment:
Nitpick so feel free to ignore but would good to do it in BDT format:

```javascript
describe('when submitting invalid characters', () => {
			it("should throw 'wrong value' api error", async () => {
```

Reply:
Sounds good to me, have updated